pom.xml

```diff
@@ -147,7 +147,7 @@
         <jakarta.el-api.version>4.0.0</jakarta.el-api.version>
         <javax.cache-api.version>1.1.1.payara-p1</javax.cache-api.version>
         <mail.version>2.0.1</mail.version>
-        <jakarta.annotation-api.version>2.0.0</jakarta.annotation-api.version>
+        <jakarta.annotation-api.version>2.1.0-B1</jakarta.annotation-api.version>
```

Comment:
it is correct this version or the 2.1.0?  I saw this last tag: https://github.com/eclipse-ee4j/common-annotations-api/releases/tag/2.1.0